ledger_app.toml

```diff
@@ -0,0 +1,3 @@
+[c-app]
+makefile-path = "./Makefile"
+tests-path = "./tests"
```

Comment:
What do you think of moving it in another section ? Like [ragger-tests]?
As rust-app are maybe also going to have ragger tests.

Reply:
Another section makes sense indeed. But `ragger-tests` feels too specific, there could be unitest, zemu tests, ...
Also I was under the impression that app mostly had a single `tests` directory, then inside `tests/functional`, `tests/unit`, ... I'm surprised boilerplate has multiple tests directories at the root of the repo. 

I propose (values specific to boilerplate):
```
[tests]
ragger-dir = "./tests/"
unit-dir = "./unit-tests"
```

Reply:
I think this [tests] section could be useful to know what kind of functional tests we have : 

* ragger (or maybe just 'pytest' ?)
* zemu (or just 'node' ?)

That would be helpful to tools that automate running those tests. 

Of course we should have a field that gives the location of the tests as well.